### ISSUE TITLE
There have been many crash times

### ISSUE THREAD

**Issue by Dong1108** [dzenbot/DZNEmptyDataSet#368]
EXC_BAD_ACCESS KERN_PROTECTION_FAILURE 0x000000016dbabfe0

UIScrollView+EmptyDataSet.m line 622
dzn_implementationKey
UIScrollView+EmptyDataSet.m line 596
dzn_original_implementation
UIScrollView+EmptyDataSet.m line 611
dzn_original_implementation

**Comment by mmdock** [comment]
@Dong1108 did you ever figure this out? I am having what looks to be a similar issue:

![screen shot 2018-06-01 at 9 56 40 am](https://user-images.githubusercontent.com/2423465/40853295-69a82c7c-6582-11e8-8b5d-97481d0b0883.png)

**Comment by colinhumber** [comment]
This library swizzles a couple of methods, namely `-reloadData`. The crash you're seeing is likely nothing with the library, but the swizzled method hiding the true cause of the crash. If you set the tableView's `emptyDataSource` and `emptyDelegate` both to `nil` to disable using this library you should see what the true cause of the crash is.

**Comment by mmdock** [comment]
@colinhumber I just removed DZNEmptyDataSet from project and used an alternative project

**Comment by colinhumber** [comment]
@mmdock Fair enough! At any rate, if you run into this again, typically the swizzled method is fine but can hide the true crash location.
